### DISCUSSION TITLE
How to go about having a feature implemented.

### DISCUSSION THREAD

**Discussion by Eldiabolo21** [netbox-community/netbox#14784] [Q&A]
Hi people, 

we are an IT-Service provider which makes heavy use of Netbox for ourselfs and our customers. 
One of them needs a new feature. They have a need for interface aggregation over multiple devices.

It has been discussed already in several issues (https://github.com/netbox-community/netbox/issues/3033,  https://github.com/netbox-community/netbox/discussions/6116 and more which I can't find) 

There were a couple of reasons for this feature no being implemented yet. One was there was just no time and ressources. 
I could image our customer paying someone to write a PR for Netbox to implement this feature. 

So how would we go about this? How can we make sure this feature will be accepted? How do we find a developer who knows Python well and ideally is familiar with the codebase of Netbox? Whats the timeline for these things?

Thanks and have a good weekend!

**Comment by abhi1693** [comment]
You can read the [contributing guide](https://github.com/netbox-community/netbox/blob/develop/CONTRIBUTING.md) which should help to answer most of your questions.

**Reply by Eldiabolo21** [reply]
Thank you for the link, that answers 1 out of the 3 questions. 
What remains open is: 
How do we find a developer who knows Python well and ideally is familiar with the codebase of Netbox? 
Whats the timeline for these things?

**Reply by abhi1693** [reply]
You first need to submit an FR as per the contributing guide, then it needs to be accepted by the maintainers. Finally, if you have someone that can do this for you, ask them to volunteer or wait until a maintainer picks it up (depending on other priorities and work queues). There is no fixed timelines on such things.
